### ISSUE TITLE
UI Forms: Event>GET/events/{id}

### ISSUE THREAD

**Issue by coultergeist** [ActoKids/AD440_W19_CloudPracticum#48]
Connect forms to corresponding API endpoints: Event>GET/events/{id}

Once assigned, annotate your estimated time. Then, keep regular updates on your issue. Comment your approach, tutorials, articles, updates, links, photos, etc. Comment often. When closing your issue include the following in your final comment.

Time estimated:
Time spent:
Pull Request link:
Wiki link:
Deployed Code link:

**Comment by hkhual** [comment]
I will do research on how to do REST API GET request on React JS. I will also do research on how to fetch API JSON data within React js. I will do research on how to request api and validate the requested data. learn to submit data via Rest API. So far I have looked at jsonplaceholder https://jsonplaceholder.typicode.com/ it is REST API that i can use to test with some fake data.  
Estimated Hour: 5
Time Spent: 6 hours

**Comment by hkhual** [comment]
Did tried json server from tutorial https://dzone.com/articles/zero-code-rest-with-json-server to mock up API. Also did Fetch API from this https://css-tricks.com/using-data-in-react-with-the-fetch-api-and-axios/ tutorial from CSS Trick. No API end point was available to test, therefore I will use json server to mock as live API.
Time spent so far: 6 Hours

**Comment by hkhual** [comment]
Here is a link a pull request. https://github.com/ActoKids/web-ui/pull/24
. Two of my team members (Zphu and Yitong ) tested the pull request and gave feedback ( looks good). You could check their feedback in the pull request. 
As of right now, we are using dummy data in place of real data since API team has not have get method created for us.

**Comment by hkhual** [comment]
![get event page](https://user-images.githubusercontent.com/24612635/53377908-8d0dc900-3918-11e9-89e1-48d397586601.JPG)
Here is the event being displayed on the dashboard. We don't have a working API to fetch data from server so we use a dumpy in json formatted. As of now, the get event display everything from the data.

**Comment by hkhual** [comment]
![fetching data from remote server](https://user-images.githubusercontent.com/24612635/53389774-716ce780-3945-11e9-96e3-2d06fe7f5f39.JPG)
using Fetch API to retrieve data from remote Server (https://jsonplaceholder.typicode.com/users) and displaying it.

**Comment by hkhual** [comment]
Here is the wiki page: https://github.com/ActoKids/web-ui/wiki/Get-Event

**Comment by hkhual** [comment]
Here is the link dashboard page. http://actokids-get.s3-website-us-east-1.amazonaws.com/

**Comment by hkhual** [comment]
Time estimated: 5 hours
Time spent: 7 hours
Pull Request link:https://github.com/ActoKids/web-ui/pull/24
Wiki link: https://github.com/ActoKids/web-ui/wiki/Get-Event
Deployed Code link: Here is the link to dashboard page. http://actokids-get.s3-website-us-east-1.amazonaws.com/ 
Here is the link to fetch api page: http://actokids-get.s3-website-us-east-1.amazonaws.com/get

**Who tested this and what is the feedback?**
Zphu and Yithong tested getEvent using fetch api. They said everything working fine. 

**What did you test and what is the feedback?**
I tested Erik branch when he was about to merge into dev branch. We got many conflicts file and different dependencies. I manually edited files causing conflicts and was about to resolved the merge conflict. And I told Erik to merge his branch into dev branch. Now everything working as expected.
